addon/models/github-release.js

```diff
@@ -18,6 +20,10 @@ export default Model.extend({
   createdAt: attr('date'),
   publishedAt: attr('date'),
 
-  user: belongsTo('github-user', { inverse: null }),
+  author: belongsTo('github-user', { inverse: null }),
+  user: computed(function() {
+    deprecate('The user property on the github-release model has been deprecated.  Please use the author property.', false, { id: 'ember-data-github.release-author', until: '1.0.0' });
```

Comment:
@elwayman02 I had a thought that we could do something like this for renaming these properties in a backwards compatible way.  If you like this I could go back and do this for those other props I renamed as well. Or we can just move forward knowing we have some minor breaking changes.

![screen shot 2018-02-12 at 07 54 04](https://user-images.githubusercontent.com/1672302/36099977-2f8094a2-0fca-11e8-8a1d-479aad9ffae8.png)

Reply:
Yea this seems like a fine way to being able to release these changes in advance. Then for v1.0 we can remove the backwards-compatible stuff.

Reply:
Perfect, I'll make another PR this week to re-add the properties I removed and deprecate them